lib/Ed25519VerificationKey2020.js

```diff
@@ -234,6 +270,57 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
     return exportedKey;
   }
 
+  /**
+   * Returns the JWK representation of this key pair.
+   *
+   * @see https://datatracker.ietf.org/doc/html/rfc8037
+   *
+   * @param {object} [options={}] - Options hashmap.
+   * @param {boolean} [options.publicKey] - Include public key?
+   * @param {boolean} [options.privateKey] - Include private key?
+   *
+   * @returns {{kty: string, crv: string, x: string, d: string}} JWK
+   *   representation.
+   */
+  toJwk({publicKey = false, privateKey = false} = {}) {
+    const jwk = {crv: 'Ed25519', kty: 'OKP'};
+    if(publicKey) {
+      jwk.x = base64url.encode(this._publicKeyBuffer);
+    }
+    if(privateKey) {
+      jwk.d = base64url.encode(this._privateKeyBuffer);
+    }
+    return jwk;
+  }
+
+  /**
+   * @see https://datatracker.ietf.org/doc/html/rfc8037#appendix-A.3
+   *
+   * @returns {Promise<string>} JWK Thumbprint.
+   */
+  async jwkThumbprint() {
+    const serialized = JSON.stringify(this.toJwk({publicKey: true}));
+    const data = new TextEncoder().encode(serialized);
+    return base64url.encode(
+      new Uint8Array(await ed25519.sha256digest({data})));
+  }
+
+  /**
+   * Returns the JsonWebKey2020 representation of this key pair.
+   *
+   * @see https://w3c-ccg.github.io/lds-jws2020/#json-web-key-2020
+   *
+   * @returns {Promise<object>} JsonWebKey2020 representation.
+   */
+  async toJsonWebKey2020() {
+    return {
+      id: this.controller + '#' + await this.jwkThumbprint(),
```

Comment:
What about `@context`?

Reply:
@dlongley I couldn't quite tell from https://w3c-ccg.github.io/lds-jws2020/ what context was required. 
Seems to be `https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json`, now that I dig through some more examples.

Reply:
Although https://github.com/perma-id/w3id.org/pull/2153/files#diff-6ee70b50cd859b83a5f0e448a8ac455bf0f6f1bc6094429df85899717f770ab9R16 seems to suggest that it's also under `w3id.org`

Reply:
(Context added.)